lib/iris/fileformats/netcdf.py

```diff
@@ -778,14 +779,17 @@ def load_cubes(filenames, callback=None):
 
     for filename in filenames:
         # Ingest the netCDF file.
-        cf = iris.fileformats.cf.CFReader(filename)
+        #         ncreader = iris.fileformats.cf.CFReader(filename)
```

Comment:
Did you mean to leave this commented-out line in? As it's proof of concept work it's fine, just checking in case you had intended to remove it

Reply:
Not really.
Will remove :wink: